cmd/ebpf/xdp/utils/gtp_utils.h

```diff
@@ -28,7 +28,7 @@
 
 static __always_inline __u32 parse_gtp(struct packet_context *ctx) {
     struct gtpuhdr *gtp = (struct gtpuhdr *)ctx->data;
-    if ((void *)(gtp + 1) > ctx->data_end)
+    if ((char *)(gtp + 1) > (char *)ctx->data_end)
```

Comment:
No convertion is needed here(and in every condition further). 
`if ((void *)(gtp + 1) > ctx->data_end)` is absolutely ok, because there are no implicit convertions.